test/functional/api/v2/compute_resources_controller_test.rb

```diff
@@ -96,6 +96,7 @@ def teardown
     assert_response :not_found
   end
 
+
```

Comment:
Remove this extra line and the other one in line 126, they'll likely cause Rubocop to fail